tests/testData/annotation/call_input_checking.wdl

```diff
@@ -0,0 +1,238 @@
+version 1.0
+
+import "nonexistent_import.wdl" as nonexistent_import
+
+workflow my_workflow {
+
+    # === NO INPUTS ===
+
+    # No inputs, none provided (correct)
+    call no_inputs
+
+    # No inputs, empty scope for input block (highlight)
+    <error descr="Task does not take inputs.">call no_inputs {
+
+    }</error>
+
+    # No inputs, empty input list (highlight)
+    # IMO: this one is a bit debatable as to whether it should highlight, since the list IS technically empty (AEN, 2018-07-17)
```

Comment:
This depends on some combination of what the spec says and what Cromwell does (need to check)